.checkov_config.yaml

```diff
@@ -11,6 +11,7 @@ quiet: true
 secrets-scan-file-type: []
 skip-check:
   - CKV_GHA_3
+  - CKV_AZURE_112
```

Comment:
I believe this is OK because not all customers require the use of HSM for Azure Key Vault